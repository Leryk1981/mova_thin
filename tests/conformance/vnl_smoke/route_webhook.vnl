@dlq.target="queues/dlq-billing"
emit channel="webhook" payload='{"route":"billing"}' idempotency_key="inv-42"

