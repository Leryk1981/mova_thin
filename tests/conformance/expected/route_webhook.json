{
  "mova_version": "3.3",
  "id": "route_webhook",
  "description": "Generated from VNL",
  "policies": {
    "timeout_ms": 15000
  },
  "actions": [
    {
      "type": "emit_event",
      "channel": "webhook",
      "payload": {
        "route": "billing"
      },
      "idempotency_key": "inv-42"
    }
  ]
}